models/public/efficientdet-d0-tf/efficientdet-d0-tf.md

```diff
@@ -0,0 +1,86 @@
+# efficientdet-d0-tf
+
+## Use Case and High-Level Description
+
+The "efficientdet-d0" model is one of the [EfficientDet](https://arxiv.org/abs/1911.09070)
+models  designed to perform object detection. This model was pretrained in TensorFlow*.
+All the EfficientDet models have been pretrained on the MSCOCO* image database.
+For details about this family of models, check out the Google AutoML [repository]
+(https://github.com/google/automl/tree/master/efficientdet).
+
+## Example
+
+## Specification
+
+| Metric            | Value           |
+|-------------------|-----------------|
+| Type              | Object detection|
+| GFLOPs            |     -           |
+| MParams           |     -           |
+| Source framework  | TensorFlow\*    |
+
+## Accuracy
+
+| Metric | Converted model |
+| ------ | --------------- |
+| coco_precision | 31.95%|
+
+## Performance
+
+## Input
+
+### Original Model
+
+Image, name - `convert_image/Cast`,  shape - `[1x512x512x3]`, format is `[BxHxWxC]`, where:
+
+- `B` - batch size
+- `H` - height
+- `W` - width
+- `C` - channel
+
+Channel order is `RGB`.
+
+### Converted Model
+
+Image, name - `convert_image/Cast/placeholder_port_0`,  shape - `[1x3x512x512]`, format is `[BxCxHxW]`, where:
+
+- `B` - batch size
+- `C` - channel
+- `H` - height
+- `W` - width
+
+Channel order is `BGR`.
+
+## Output
+
+### Original Model
+
+The array of summary detection information, name: `detections`, shape: [1, 7, N], where N is the number of detected
```

Comment:
is it really has SSD Detection Output in source framework???

Reply:
Yes, except the order is different

Reply:
There is something wrong here. We designed a model to produce 1x1xNx7 as all the SSDs. See https://github.com/openvinotoolkit/openvino/pull/2894 and https://github.com/openvinotoolkit/openvino/pull/2691

/cc @lazarevevgeny

Reply:
@dkurt, the shape order was incorrect. Original model produces 1xNx7 output ([inference.py#L241](https://github.com/google/automl/blob/341af7d4da7805c3a874877484e133f33c420ec5/efficientdet/inference.py#L241)) Fixed